start-job/scripts/zalando.py

```diff
@@ -17,16 +17,13 @@ def combine_results(
     results = []
     for path, info in path_2_category.items():
         category, meta_data = info if type(info) == tuple else (info, {})
-        for filter in filters:
-            results.append(
-                {
-                    "start_urls": f"https://www.zalando.de/{path}/?cause={filter}",
-                    "category": category,
-                    "meta_data": json.dumps(
-                        {"family": "FASHION", "sustainability": filter, "sex": sex, **meta_data}
```

Comment:
By sacrificing `"sustainability": filter` as part of a products meta data, we can also set directly all filters instead of each category with every filter. This reduces the number of zalando spiders by a factor of about 5 (times 2 because we have 2 zalando spiders (DE/FR)).

This should avoid scraping the same product multiple times and reduce the overall time.